tck/bridge-tck-main-portlet/test-liferay62-glassfish.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-mvn surefire-report:report -P tck,liferay,liferay62,glassfish
+mvn verify -P selenium,liferay62,chrome,glassfish "$@"
```

Comment:
@stiemannkj1 no, it's not necessary.  But, it's consistent with the other `liferay70` scripts. Also, is `chrome` always going to be the default browser? If so, I can remove the `chrome` profile for all the scripts.